### ISSUE TITLE
Remove warning with django 3.2;X

### ISSUE THREAD

**Issue by MiStErLu** [mbi/django-simple-captcha#205]
Hi,

With the version of Django 3.2 we have a warning when we start.

```
WARNINGS:
captcha.CaptchaStore: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
```

In order to fix that, we just need to add in the CaptachStore model this entry : 
    
`id = models.AutoField(primary_key=True)`

**Comment by mbi** [comment]
LGTM. Can you please open a PR? We need to test this with all supported Django versions, i.e. 2.2+

Thanks!

**Comment by MiStErLu** [comment]
Hi,

   I try to create a branch in order to make a pull request but i do probably something wrong. How can i do that ?

Best regards,
MiStErLu.

**Comment by mbi** [comment]
Here you go: https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

You could also just just edit https://github.com/mbi/django-simple-captcha/blob/master/captcha/models.py directly by clicking the pen icon, apply your change and then create a PR when prompted how to save the change.

Thanks :pray:

**Comment by MiStErLu** [comment]
Thanks for you're help :-)

**Comment by mbi** [comment]
Merged via d3908b42c4acc54171ca2627c6583ea5a4ec3f7f, thanks!
